app/models/discussion.rb

```diff
@@ -13,8 +13,6 @@ class Discussion < ActiveRecord::Base
   private
 
   def update_subscriptions
-    User.all.each do |user|
-      Subscription.for(discussion_id: id, user_id: user.id).new_comment
-    end
+    Subscriber.for_discussion(self).subscribe(User.all)
```

Comment:
I don't think this is doing the same thing. I was calling the `new_comment` method above (which I know, is not obvious given the name of this method. that part wasn't so good, but I was also in the process of refactoring)